### ISSUE TITLE
 Custom keyboard navigation in IE 11

### ISSUE THREAD

**Issue by randspy** [ng-bootstrap/ng-bootstrap#4147]
### Bug description:

Custom keyboard navigation does not work in IE. In specified example Alt + Page Up/Down should change a year but it changes a month.

### Link to minimally-working StackBlitz that reproduces the issue:

https://ng-bootstrap.github.io/#/components/datepicker/examples#keyboard
